roles/georchestra/templates/mviewerstudio/mviewerstudio.service.j2

```diff
@@ -13,9 +13,9 @@
  WorkingDirectory=/srv/apps/mviewerstudio/srv/python
  ExecStart=/srv/apps/mviewerstudio/srv/python/.venv/bin/gunicorn \
      -b 127.0.0.1:{{ mviewerstudio.port}} \
-     --access-logfile /{{ logs_basedir }}/mviewerstudio-access.log \
+     --access-logfile /{{ logs_basedir }}/mviewer/mviewerstudio-access.log \
```

Comment:
still extra leading slash, eg this ends up as `--access-logfile //srv/log/mviewer/mviewerstudio-access.log`